openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -32,10 +32,13 @@ if (typeof OpenAssessment.unsavedChanges === 'undefined' || !OpenAssessment.unsa
     OpenAssessment.unsavedChanges = {};
 }
 
+var usageID;
```

Comment:
Please define usageID above in the constructor, instead of defining it outside of OpenAssessment.BaseView.